graphics/gegl3/Makefile

```diff
@@ -79,7 +79,7 @@ JASPER_CONFIGURE_WITH=	jasper
 LCMS2_LIB_DEPENDS=	liblcms2.so:graphics/lcms2
 LCMS2_CONFIGURE_OFF=	--without-lcms
 
-LUA_USES=		lua:5.2
+LUA_USES=		lua
```

Comment:
IIRC this was an intentional patch, I upgraded the Lua version for a reason (not that I remember *what* reason), please don't roll back to 5.1

Reply:
OK done.